src/webview/html/messageAsHtml.js

```diff
@@ -43,6 +43,15 @@ const messageReactionAsHtml = (
       : codeToEmojiMap[reaction.code]
   }&nbsp;${reaction.count}</span>`;
 
+const addReactionButtonAsHtml = (): string =>
+  template` <span class="add-emoji-button" 
+      data-name="plus"
+      data-code="2795"
+      data-type="unicode_type">
+      ${codeToEmojiMap['2795']}
```

Comment:
This is going to just be the string `➕`, so it'd be clearer to write that directly.

Or you can write it with an escape sequence (part of JS's string-literal syntax): `\u{2795}`. In that case there should be a comment with the actual character. But I think simplest is to have the actual character.

Reply:
Done :)